crates/routes/src/feeds.rs

```diff
@@ -150,21 +150,20 @@ async fn get_feed_data(
 }
 
 async fn get_feed_user(
-  req: HttpRequest,
   info: web::Query<Params>,
+  name: web::Path<String>,
   context: web::Data<LemmyContext>,
 ) -> Result<HttpResponse, Error> {
-  let user_name: String = req
-    .match_info()
-    .get("user_name")
-    .unwrap_or("none")
-    .parse()?;
+  let (name, domain) = split_name(name.into_inner());
```

Comment:
You can't call into_inner here, you still need to keep the parsing like before to get the correct field.

Reply:
Of course I can: https://actix.rs/docs/extractors#path